packages/client-cli/test/cli-openapi-additional-properties.test.mjs

```diff
@@ -0,0 +1,21 @@
+import { moveToTmpdir } from './helper.js'
+import { test, after } from 'node:test'
+import { equal } from 'node:assert'
+import { join } from 'path'
+import * as desm from 'desm'
+import { execa } from 'execa'
+import { readFile } from 'fs/promises'
+
+test('export formdata on full request object', async (t) => {
+  const dir = await moveToTmpdir(after)
+
+  const openAPIfile = desm.join(import.meta.url, 'fixtures', 'additional-properties-openapi.json')
+  await execa('node', [desm.join(import.meta.url, '..', 'cli.mjs'), openAPIfile, '--name', 'additional-props', '--full-request', '--full-response'])
+  const typeFile = join(dir, 'additional-props', 'additional-props.d.ts')
+  const data = await readFile(typeFile, 'utf-8')
+  equal(data.includes(`
+  export type GetSampleResponseOK = {
+    'entities': { 'foo'?: { 'id'?: string; 'name'?: string }; 'bar'?: { 'type'?: 'boolean' | 'list'; 'values'?: Array<{ 'id': string; 'isArchived'?: boolean; 'isDefault': boolean; 'value': string }> }; 'baz'?: { 'id'?: string; 'name'?: string } };
```

Comment:
I think this should be instead:

```ts
{
  entities: {
    foo?: Record<string, { 'id'?: string; 'name'?: string }>;
    // or 
    foo?: { [key: string]: { 'id'?: string; 'name'?: string } };
...
```

Reply:
For comparison, this is what `openapi-ts` generates:

```ts
entities: {
  foo?: {
      [key: string]: {
          id?: string;
          name?: string;
      };
  };
```

Reply:
good point, fixed on last commit 6ec7335